exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py

```diff
@@ -84,10 +84,7 @@ def __init__(
         timeout: Optional[int] = None,
         compression: Optional[Compression] = None,
     ):
-        if (
-            not insecure
-            and environ.get(OTEL_EXPORTER_OTLP_TRACES_CERTIFICATE) is not None
-        ):
+        if environ.get(OTEL_EXPORTER_OTLP_TRACES_CERTIFICATE) is not None:
```

Comment:
Should `insecure` not be considered to set the credentials here? It seems like the logic to define `insecure` in the mixin should be used here as well.

Reply:
Makes sense, it was changing previous behavior to remove the check. I have added it back in with the mixin logic.